packages/react-reconciler/src/__tests__/ReactNewContext-test.internal.js

```diff
@@ -711,18 +709,15 @@ describe('ReactNewContext', () => {
     ReactNoop.render(<Context.Provider value={1} />);
     ReactNoop.flush();
 
-    // Update
-    ReactNoop.render(<Context.Provider value={2} />);
-    ReactNoop.flush();
-
-    if (__DEV__) {
-      expect(console.error).toHaveBeenCalledTimes(1);
-      expect(console.error.calls.argsFor(0)[0]).toContain(
-        'calculateChangedBits: Expected the return value to be a 31-bit ' +
-          'integer. Instead received: 4294967295',
-        {withoutStack: true}, // TODO: add a stack
-      );
-    }
+    expect(() => {
+      // Update
+      ReactNoop.render(<Context.Provider value={2} />);
+      ReactNoop.flush();
+    }).toWarnDev(
```

Comment:
tiny nit: Might be nicer to move `ReactNoop.render` outside of the `expect` call.